packages/slate-sections-plugin/index.js

```diff
@@ -0,0 +1,220 @@
+const fs = require('fs-extra');
+const path = require('path');
+const {ConcatSource, RawSource} = require('webpack-sources');
+const _ = require('lodash');
+
+module.exports = class sectionsPlugin {
+  constructor(options) {
+    this.options = options;
+  }
+  apply(compiler) {
+    compiler.hooks.emit.tapAsync(
+      'Slate Sections Plugin',
+      this.addLocales.bind(this),
+    );
+  }
+
+  async addLocales(compilation, callback) {
+    const files = await fs.readdir(this.options.from);
+    await Promise.all(
+      files.map(async (file) => {
+        const fileStat = await fs.stat(path.resolve(this.options.from, file));
+        if (fileStat.isDirectory()) {
+          const pathToLiquidFile = path.resolve(
+            this.options.from,
+            file,
+            this.options.genericTemplateName,
+          );
+          const outputKey = this._getOutputKey(pathToLiquidFile, compilation);
```

Comment:
You could declare up top `const compilationOutput = compilation.compiler.outputPath` and pass it here